Makefile

```diff
@@ -1,5 +1,11 @@
 # Current Operator version
-VERSION ?= 0.0.1
+VERSION_MAJOR=0
+VERSION_MINOR=1
+COMMIT_NUMBER=$(shell git rev-list `git rev-list --parents HEAD | egrep "^[a-f0-9]{40}$$"`..HEAD --count)
+CURRENT_COMMIT=$(shell git rev-parse --short=7 HEAD)
+OPERATOR_VERSION=$(VERSION_MAJOR).$(VERSION_MINOR).$(COMMIT_NUMBER)-$(CURRENT_COMMIT)
+
+VERSION ?= $(OPERATOR_VERSION)
```

Comment:
This is only used once in `148`. Why not using there `OPERATOR_VERSION` directly?

Reply:
I wanted to keep the codebase (which was copied from operator-sdk) the same as much as I can. if we really want to change it I can live with that